streamwebs_frontend/streamwebs/templates/streamwebs/sites.html

```diff
@@ -9,14 +9,15 @@
 <script type="application/javascript">
     var sites = [];
     {% for site in sites %}
-    sites.push({
-        slug: "{{site.slug}}",
-        name: "{{site.name}}",
-        type: "{{site.site_type}}",
-        description: "{{site.description}}",
-        lat: {{site.location.y}},
-        lng: {{site.location.x}},
-    });
+        sites.push({
+            id:   "{{site.id}}",
+            slug: "{{site.site_slug}}",
+            name: "{{site.site_name}}",
+            type: "{{site.site_type}}",
+            description: "{{site.description}}",
+            lat: {{site.location.y}},
```

Comment:
The coordinate assignments are mixed up. Latitude is actually stored as site.location.x in the db and longitude as y.